coreforge/geometry_elements/cylindrical_pincell.py

```diff
@@ -100,26 +100,28 @@ def mpact_build_specs(self) -> Optional[MPACTBuildSpecs]:
 
     @mpact_build_specs.setter
     def mpact_build_specs(self, specs: Optional[MPACTBuildSpecs]) -> None:
-        if specs:
-            self._radial_thicknesses = []
-            prev_radius = 0.
-            for zone in self.zones:
-                self._radial_thicknesses.append(zone.shape.inner_radius - prev_radius)
-                prev_radius = zone.shape.inner_radius
-
-            self._module_specs = [{"bounds" : specs.bounds}]
-            if specs.divide_into_quadrants:
-                xmin = specs.bounds[0]
-                xmax = specs.bounds[1]
-                ymin = specs.bounds[2]
-                ymax = specs.bounds[3]
-                hp = {"X": (xmax-xmin)*0.5, "Y": (ymax-ymin)*0.5} # half pitch
-                self._module_specs = [{"bounds" : (        xmin, xmin+hp["X"], ymin+hp["Y"],         ymax)},
-                                      {"bounds" : (xmin+hp["X"],         xmax, ymin+hp["Y"],         ymax)},
-                                      {"bounds" : (        xmin, xmin+hp["X"],         ymin, ymin+hp["Y"])},
-                                      {"bounds" : (xmin+hp["X"],         xmax,         ymin, ymin+hp["Y"])}]
-
-        self._mpact_build_specs = specs
+        outer_radius = self.zones[-1].shape.outer_radius
+        self._mpact_build_specs = specs if specs else \
+                                  CylindricalPincell.MPACTBuildSpecs(bounds=(-outer_radius, outer_radius
+                                                                             -outer_radius, outer_radius))
```

Comment:
```suggestion
     CylindricalPincell.MPACTBuildSpecs(
          bounds=(
                -outer_radius,
                 outer_radius,
                -outer_radius,
                 outer_radius,
          )
     )
```
I think the poor formatting is hiding a bug.  There is a missing comma (I think).  Also, this piece of the code must not be covered with a unit test, or it likely would've shown itself.

Reply:
Fixed.  Yeah, the test specifies the bounds so the default was not exercised.  Added test to flex this part of the code.